iron-menu-behavior.html

```diff
@@ -117,19 +118,27 @@
       }
     },
 
-    /**
-     * Given a KeyboardEvent, this method will focus the appropriate item in the
-     * menu (if there is a relevant item, and it is possible to focus it).
-     *
-     * @param {KeyboardEvent} event A KeyboardEvent.
-     */
-    _focusWithKeyboardEvent: function(event) {
+    _clearSearchText: function() {
+      this._searchText = '';
+    },
+    
+    _searchForMatchingItem: function() {
+      var searchText = this._searchText;
+      var searchLength = searchText.length;
+      
       for (var i = 0, item; item = this.items[i]; i++) {
+        if (item.hasAttribute('disabled')) {
+          continue;
+        }
+        
         var attr = this.attrForItemTitle || 'textContent';
-        var title = item[attr] || item.getAttribute(attr);
+        var title = (item[attr] || item.getAttribute(attr)).trim();
 
-        if (!item.hasAttribute('disabled') && title &&
-            title.trim().charAt(0).toLowerCase() === String.fromCharCode(event.keyCode).toLowerCase()) {
+        if (title.length < searchLength) {
+          continue;
+        }
+        
+        if (title.slice(0, searchLength).toLowerCase() == searchText) {
```

Comment:
I think the change you recommended earlier (`toLowerCase` -> `toLocaleLowerCase`) is a good idea.